tools/test/audio/process_test.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+# SPDX-License-Identifier: BSD-3-Clause
+# Copyright(c) 2020 Intel Corporation. All rights reserved.
+
+set -e
+
+# This process_test.sh is shell script wrapper for process_test.m
+# This needs octave octave-signal octave-io packages.
```

Comment:
This new, tiny `.sh` file is used only once inside a much bigger and already existing `.sh` file. Why not just a new shell function?

Reply:
thank you for the suggestion